lib/tasks/deployment/20221009032756_add_medium_type_to_howard_court_report_template.rake

```diff
@@ -0,0 +1,13 @@
+namespace :after_party do
+  desc "Deployment task: add_medium_type_to_howard_court_report_template"
+  task add_medium_type_to_howard_court_report_template: :environment do
+    puts "Running deploy task 'add_medium_type_to_howard_court_report_template'"
+
+    CasaOrg.where(name: "Howard County CASA").map do |casa_org|
```

Comment:
Not 100% sure if this is the correct name here. Let me know if I need to update it!